### PR TITLE
Editorial pass on BigAnimal content

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

Comprehensive editing pass on the BigAnimal content

### CODE REVIEW

**Comment by drothery-edb** [comment]
Rebased

File: product_docs/docs/biganimal/release/overview/05_database_version_policy.mdx [hunk 1]
```diff
@@ -7,19 +7,19 @@ title: "Database version policy"
...
 
-PostgreSQL and EDB Postgres Advanced Server major versions are supported from the date they are made available until the version is retired by EDB (generally 5 years). See [Platform Compatibility ](https://www.enterprisedb.com/platform-compatibility#epas) for more details.
+PostgreSQL and EDB Postgres Advanced Server major versions are supported from the date they are made available until the version is retired by EDB (generally five years). See [Platform Compatibility ](https://www.enterprisedb.com/platform-compatibility#epas) for more details.
 
 
 
 ## Minor version support
 
-EDB performs periodic maintenance to ensure stability and security. EDB automatically performs minor version upgrades and patch updates as part of periodic maintenance. Customers are notified within the BigAnimal portal prior to maintenance occurring. Minor versions are not user configurable. 
+EDB performs periodic maintenance to ensure stability and security. EDB performs minor version upgrades and patch updates as part of periodic maintenance. Customers are notified in the BigAnimal portal prior to maintenance occurring. You cannot configure minor versions.
```
Comment (dwicinas) [inline product_docs/docs/biganimal/release/overview/05_database_version_policy.mdx:22]: I know we frown on adverbs, but I think "automatically" provides some value in this paragraph.

File: product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx [hunk 2]
```diff
@@ -2,14 +2,14 @@
 title: "Responsibility model"
 ---
 
-Security and confidentiality is a shared responsibility between you and EDB. EDB provides a secure platform that enables you to create and maintain secure database clusters deployed on BigAnimal. You have numerous responsibilities around the security of your clusters and data held within them.
+Security and confidentiality are a shared responsibility between you and EDB. EDB provides a secure platform that enables you to create and maintain secure database clusters deployed on BigAnimal. You have severl responsibilities around the security of your clusters and the data they contain.
```
Comment (moiznalwalla) [inline product_docs/docs/biganimal/release/overview/04_responsibility_model.mdx]: Should this be "several"?

File: product_docs/docs/biganimal/release/getting_started/01_check_resource_limits.mdx [hunk 3]
```diff
@@ -4,35 +4,35 @@ title: Raising Azure resource limits
...
 
-The default Public IP address limits for Public IP Addresses Basic and Public IP Addresses Standards is set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information. You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` for the regions where you plan to deploy your EDB clusters with the total number of EDB clusters you plan to use.
+The default Public IP address limits for Public IP Addresses Basic and Public IP Addresses Standards is set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information. You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` with the total number of EDB clusters you plan to use for the regions where you plan to deploy your EDB clusters.
```
Comment (moiznalwalla) [inline product_docs/docs/biganimal/release/getting_started/01_check_resource_limits.mdx:9]: The original statement was inaccurate to begin with, but i think we can simplify as follows:

"You need to increase the limit of `Public IP Addresses - Basic` and `Public IP Addresses - Standard` to the total number of clusters you plan to deploy in a particular region."

File: product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx [hunk 4]
```diff
@@ -2,39 +2,37 @@
 title: "Connecting your BigAnimal account"
 ---
 
-This topic describes how to set up your BigAnimal account on Azure Marketplace. Your Azure subscription for BigAnimal is where you create and manage Postgres clusters.
+Set up your BigAnimal account on Azure Marketplace. Your Azure subscription for BigAnimal is where you create and manage Postgres clusters.
```
Comment (moiznalwalla) [inline product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx:5]: Based on a discussion with Ben and Dee Dee we decided to replace all references to clusters (such as Postgres clusters, EDB clusters, and so on) with the term "clusters".

File: product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/06_metrics.mdx [hunk 5]
```diff
@@ -60,63 +57,62 @@ BigAnimal collects a wide set of metrics about Postgres instances into the
...
-KQL can be used to analyze time-series metrics, report latest samples of
-metrics, etc by querying the `InsightsMetrics` table.
+You can use KQL to analyze time-series metrics, report latest samples of
+metrics, and so on by querying the `InsightsMetrics` table.
 
-Some data from Postgres monitoring system views, tables and functions are
+Some data from Postgres monitoring system views, tables, and functions are
 transformed to be easier to consume in Prometheus metrics format. For example,
-timestamp fields are generally converted to unix epoch time and/or accompanied
+timestamp fields are generally converted to Unix epoch time and can be accompanied
```
Comment (moiznalwalla) [inline product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/06_metrics.mdx:65]: should Unix be capitalised to UNIX?

File: product_docs/docs/biganimal/release/administering_cluster/index.mdx [hunk 6]
```diff
@@ -2,4 +2,4 @@
 title: "Administering your account"
 ---
 
-In this section, account owners can find information on administrative activities for the BigAnimal account including portal and database user access management as well as account activity reviews.
+Administrative activities for the BigAnimal account includ portal and database user access management as well as account activity reviews.
```
Comment (moiznalwalla) [inline product_docs/docs/biganimal/release/administering_cluster/index.mdx]: "include"

File: product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx (hunk 4 above)
Comment (moiznalwalla) [reply product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx:5]: Here is a link to that conversation: https://github.com/EnterpriseDB/docs/pull/1855#discussion_r731114575

**Comment by moiznalwalla** [comment]
Another discussion we had was Ben was around capitalisation of some the Azure's resources which were even inconsistently capitalised in their own docs. Would love to hear your opinion on those. I cant seem to find that comment/discussion but its likely to come up in one of the other topics I am working on on Azure Security. I'll keep you in loop.

**Comment by josh-heyer** [comment]
Building a draft deploy - gonna run link checker on this before we merge

**Review by josh-heyer — APPROVED** [review]
Fixed a couple of links, should be gtg!

**Pull request merged**